docs/cli/makei.md

```diff
@@ -140,6 +140,14 @@ It is likely that the same destination directory will contain converted members
 
   An optional CCSID for the SRC-PF. See the description above for more details.
```

Comment:
An optional CCSID to use as TGTCCSID on compiles if the SRC-PF CCSID being migrated is 65535. See above for in-depth description.